game/server_core.cpp

```diff
@@ -87,15 +88,49 @@ void ServerCore::receive_data()
     }
 }
 
-void ServerCore::process_input() {}
+void ServerCore::process_input(InputPacket packet) {
+    // For now operate on first player by default. TODO: Identify by socket num? Client ID?
+    glm::mat4 world = serverState.players[0].world;
```

Comment:
ServerCore::receive_data iterates through clients with a foreach loop rn, but it could easily be switched to a `for i < clients_data.size()` type condition... Then we can pass the index of the current client to process_input to identify which client sent that input.